src/frontend/stores/UiStore.ts

```diff
@@ -345,6 +348,16 @@ class UiStore {
     }
   }
 
+  @action.bound openExternal() {
+    // Don't open when no files have been selected
+    if (this.fileSelection.size === 0) {
+      return;
+    }
+
+    const absolutePaths = Array.from(this.fileSelection, (file) => file.absolutePath);
+    absolutePaths.forEach((path) => shell.openExternal(`file://${path}`).catch(console.error));
```

Comment:
as much as I don't like arbitrary hardcoded limits, it might be good to set one here. We don't any anyone's computer to explode when they have hundreds or thousands of images selected. 
Does 25 or so seem reasonable to you?